### PR TITLE
Memory alloc fixes for nanosvg and easyexif

### PR DESCRIPTION
**Pull request by SmilyOrg**

### CODE REVIEW

**Comment by SmilyOrg** [comment]
Please wait for #405 to land first, thanks!

**Comment by bengarney** [comment]
This needs a rebase, can you do so?

**Comment by SmilyOrg** [comment]
Will do in a bit.

**Comment by SmilyOrg** [comment]
Rebased and "minor" fix? :)

File: loom/graphics/gfxVectorRenderer.cpp [hunk 1]
```diff
@@ -705,7 +705,8 @@ void VectorRenderer::reset()
 
 void VectorRenderer::initialize()
 {
-    nvgSetAllocFunctions(nvgAlloc, nvgRealloc, nvgFree);
+    nvgSetAllocFunctions(customAlloc, customRealloc, customFree);
+    nsvgSetAllocFunctions(customAlloc, customRealloc, customFree);
```
Comment (bengarney) [inline loom/graphics/gfxVectorRenderer.cpp]: I was going to complain about this line but I see now it's svg vs vg. Good catch.

**Comment by bengarney** [comment]
OK this looks good. Cleared to land when green.

**Comment by SmilyOrg** [comment]
:green_apple:

**Pull request merged**